Utils/TimeConstants.swift

```diff
@@ -18,6 +18,45 @@ extension NSDate {
     public class func now() -> Timestamp {
         return UInt64(1000 * NSDate().timeIntervalSince1970)
     }
+    
+    public func toRelativeTimeString() -> String {
+        let now = NSDate()
+        let units = NSCalendarUnit.CalendarUnitSecond | NSCalendarUnit.CalendarUnitMinute | NSCalendarUnit.CalendarUnitDay | NSCalendarUnit.CalendarUnitWeekOfYear | NSCalendarUnit.CalendarUnitMonth | NSCalendarUnit.CalendarUnitYear | NSCalendarUnit.CalendarUnitHour
+        let components = NSCalendar.currentCalendar().components(units, fromDate: self, toDate: now, options: NSCalendarOptions.allZeros)
+        let formatter = NSDateFormatter()
+        
+        if components.year > 0 {
+            return String(format: NSLocalizedString("%@", comment: "Relative date for date older than an year."), NSDateFormatter.localizedStringFromDate(self, dateStyle: NSDateFormatterStyle.ShortStyle, timeStyle: NSDateFormatterStyle.ShortStyle))
+        }
+
+        if components.month > 0 {
+            if components.month == 1 {
+                return String(format: NSLocalizedString("More than a month ago", comment: "Relative date for dates older than a month and less than two months."))
+            }
+            else {
+                return String(format: NSLocalizedString("%@", comment: "Relative date for date older than two months."), NSDateFormatter.localizedStringFromDate(self, dateStyle: NSDateFormatterStyle.ShortStyle, timeStyle: NSDateFormatterStyle.ShortStyle))
```

Comment:
Again, this isn't a relative date; it's a short-form absolute, so you don't need a localized string.

Whenever you see `NSLocalizedString("%@"`, you can simplify.